src/Sentry/Internal/ReadOnlyFilesystem.cs

```diff
@@ -2,20 +2,24 @@ namespace Sentry.Internal;
 
 internal class ReadOnlyFileSystem : FileSystemBase
 {
-    public override FileOperationResult CreateDirectory(string path) => FileOperationResult.Disabled;
+    // Note: You are responsible for handling success/failure when attempting to write to disk.
```

Comment:
this won't show up in the IDE at call site. Needs <summary>

Reply:
These interfaces and classes are all internal, so it looks like the only place to document this is in some remarks or something on `SentryOptions.DisableFileWrite`.